ReactWindows/ReactNative/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -0,0 +1,461 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.Reflection;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using ReactNative.Views.Text;
+using System.Collections.Generic;
+using Windows.System;
+using Windows.UI;
+using Windows.UI.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+
+namespace ReactNative.Views.TextInput
+{
+    class ReactPasswordBoxManager : BaseViewManager<PasswordBox, ReactPasswordBoxShadowNode>
+    {
+        private const int FocusTextInput = 1;
```

Comment:
>private [](start = 8, length = 7)

Since you're reusing the TextInputState, you should make the command IDs public on ReactTextInputManager and use those here, rather than re-declaring them.